react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -188,18 +204,28 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
         }
     }
 
-    private func shouldLog(with level: Int) -> Bool {
-        return level >= RCTGetLogThreshold().rawValue
+    public func connectionEstablished() {
+        guard !hasObservers else { return } // We have an established connection no need to continue.
+        hasObservers = true
```

Comment:
`hasObservers` and `isJSLoading` seems to be doing pretty much the same. Not sure if it's good idea to refactor everything on this same PR though. 

I'm happy to have a follow up PR with these changes.

Reply:
I agree this would be a good idea. I started going down this path in this PR but as it was really focused on making sure this pattern worked I was hesitant to make all of those changes here. 

I just opened up a new issue to track this debit https://github.com/wordpress-mobile/gutenberg-mobile/issues/2390

Reply:
Amazing! Thank you 🙏